3.0.0/reference.json

```diff
@@ -2238,6 +2238,13 @@
                 "doc": "Rotate the text. (only works with text-placement:point).",
                 "default-meaning": "Text is not rotated and is displayed upright."
             },
+            "rotate-displacement": {
+                "css": "text-rotate-displacement",
+                "type": "boolean",
+                "doc": "Rotates the displacement around the placement origin by the angle given by \"orientation\".",
+                "default-value": false,
+                "default-meaning": ""
```

Comment:
Can you suggest a `default-meaning` so I can merge?
Could be something like "Label center is used for rotation."
Thanks :)